php/commands/src/CLI_Command.php

```diff
@@ -575,4 +575,37 @@ private function get_update_type_str( $assoc_args ) {
 		return $update_type;
 	}
 
+	/**
+	 * Detects if a command exists
+	 *
+	 * This commands checks if a command is registered with WP-CLI.
+	 * If the command is found then it returns with exit status 0.
+	 * If the command doesn't exist, then it will exit with status 1.
+	 *
+	 * ## OPTIONS
+	 * <command_name>...
+	 * : The command
+	 *
+	 * ## EXAMPLES
+	 *     wp cli has-command "wp site delete"
+	 *     wp cli has-command wp site delete
+	 *
+	 * @subcommand has-command
+	 *
+	 * @when after_wp_load
+	 *
+	 * @param array $_          Array of positional arguments.
+	 * @param array $assoc_args Array of associative arguments.
```

Comment:
We should remove this argument definition, as it's not a pattern we have elsewhere.